src/app/modules/teacher/models/time-stamp-from-csharp.ts

```diff
@@ -0,0 +1,29 @@
+export class TimeStampFromCSharp {
```

Comment:
Do we really need this or can we use https://www.javatpoint.com/typescript-date-object#:~:text=The%20Date%20object%20represents%20a,time%20of%20the%20user's%20computer. with some configuration from the backend?